app/models/user.rb

```diff
@@ -342,14 +342,29 @@ def destroy
     super
   end
 
-  %w[veteran_status military_information payment].each do |emis_method|
-    define_method(emis_method) do
-      emis_model = instance_variable_get(:"@#{emis_method}")
-      return emis_model if emis_model.present?
-
-      emis_model = "EMISRedis::#{emis_method.camelize}".constantize.for_user(self)
-      instance_variable_set(:"@#{emis_method}", emis_model)
-      emis_model
+  military_information_method = 'military_information'
+  define_method(military_information_method) do
+    model = instance_variable_get(:"@#{military_information_method}")
```

Comment:
Can we also remove the meta programming from the rest of this function?

`return @military_information if @military_information.present?`, etc

Reply:
@rjohnson2011 see comment above about me handling this in the military_information work I'm doing.